### PR TITLE
Homework 2 completed

### PR DESCRIPTION
**Pull request by serafimovmax**
Дмитрий, привет!
Закончил вторую домашку. 
Очень понравилось её делать.

### CODE REVIEW

File: src/test/resources/suite.epam.tc.hw2/HomePageOneTestSuite [hunk 1]
```diff
@@ -0,0 +1,8 @@
+<!DOCTYPE suite SYSTEM "https://testng.org/testng-1.0.dtd" >
+<suite name="Home Page One Test Suite">
+  <test name="Home Page One">
+    <classes>
+      <class name="com.epam.tc.hw2.ex1.HomePageOneTest"/>
+    </classes>
+  </test>
+</suite>
```
Comment (komyagin) [inline src/test/resources/suite.epam.tc.hw2/HomePageOneTestSuite:8]: Add a new blank line. Did you run your project via "mvn test"?

File: src/test/resources/suite.epam.tc.hw2/HomePageTwoTestSuite [hunk 2]
```diff
@@ -0,0 +1,8 @@
+<!DOCTYPE suite SYSTEM "https://testng.org/testng-1.0.dtd" >
+<suite name="Home Page Two Test Suite">
+  <test name="Home Page Two">
+    <classes>
+      <class name="com.epam.tc.hw2.ex2.HomePageTwoTest"/>
+    </classes>
+  </test>
+</suite>
```
Comment (komyagin) [inline src/test/resources/suite.epam.tc.hw2/HomePageTwoTestSuite:8]: Add a new blank line. Did you run your project via "mvn test"?

**Review by komyagin — CHANGES_REQUESTED** [review]

File: src/test/java/com/epam/tc/hw2/BaseHomePageTest.java [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+    @BeforeClass(alwaysRun = true)
+    public void setUp() {
+        WebDriverManager.chromedriver().setup();
+        webDriver = new ChromeDriver();
+        webDriver.manage().window().maximize();
+        webDriver.manage().timeouts()
+                 .implicitlyWait(10L, TimeUnit.SECONDS);
+
+        //1. Open test site by URL
+        webDriver.navigate().to("https://jdi-testing.github.io/jdi-light/index.html");
```
[the remark this answers is not in the document]
Comment (serafimovmax) [reply src/test/java/com/epam/tc/hw2/BaseHomePageTest.java:25]: Done

File: src/test/java/com/epam/tc/hw2/BaseHomePageTest.java [hunk 4]
```diff
@@ -0,0 +1,45 @@
...
+
+        //2. Assert Browser title
+        softAssert.assertEquals(webDriver.getTitle(), "Home Page");
```
[the remark this answers is not in the document]
Comment (serafimovmax) [reply src/test/java/com/epam/tc/hw2/BaseHomePageTest.java:28]: Done

File: src/test/resources/suite.epam.tc.hw2/HomePageOneTestSuite (hunk 1 above)
Comment (serafimovmax) [reply src/test/resources/suite.epam.tc.hw2/HomePageOneTestSuite:8]: The blank line is added. Yes I did. The test is running successfully.

File: src/test/resources/suite.epam.tc.hw2/HomePageTwoTestSuite (hunk 2 above)
Comment (serafimovmax) [reply src/test/resources/suite.epam.tc.hw2/HomePageTwoTestSuite:8]: The blank line is added. Yes I did. The test is running successfully.

File: .gitignore [hunk 5]
```diff
@@ -1,4 +1,7 @@
 target
 *.iml
 .idea
-.DS_Store
\ No newline at end of file
+.DS_Store
+*.DS_Store
+*/.DS_Store
+**/.DS_Store
```
Comment (komyagin) [inline .gitignore:7]: Add a new blank line

File: src/test/java/com/epam/tc/hw2/BaseHomePageTest.java [hunk 6]
```diff
@@ -22,20 +22,20 @@ public void setUp() {
                  .implicitlyWait(10L, TimeUnit.SECONDS);
 
         //1. Open test site by URL
-        webDriver.navigate().to("https://jdi-testing.github.io/jdi-light/index.html");
+        webDriver.navigate().to(Constants.HOME_PAGE.get());
```
Comment (komyagin) [inline src/test/java/com/epam/tc/hw2/BaseHomePageTest.java:25]: If you use from Constants so often why don't you use import for Constants?

File: src/test/resources/suite.epam.tc.hw2/HomePageOneTestSuite (hunk 1 above)
Comment (komyagin) [reply src/test/resources/suite.epam.tc.hw2/HomePageOneTestSuite:10]: I told about a new line at the end of the file. Don't you run your code through the "mvn test" command?

**Review by komyagin — CHANGES_REQUESTED** [review]

**Review by komyagin — APPROVED** [review]

**Pull request merged**